scss/_patterns_lists.scss

```diff
@@ -64,33 +64,23 @@ $list-status-icon-nudge: 0.3rem; // 0.3rem seems to be a magic number that posit
 
   %vf-list-item-state-base {
     @extend %vf-list;
-
-    background-position-y: $spv--x-small + $list-status-icon-nudge;
     background-repeat: no-repeat;
     background-size: $list-status-icon-height;
```

Comment:
Are the background props still needed on this one?
My guess is these were needed when ticked icon was on item itself, but now we moved it into `:before`, right?